app/templates/services/service_submission.html

```diff
@@ -7,32 +7,40 @@
 {% import "macros/submission.html" as submission %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {% with items = [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
+        "text": "Your account",
+        "href": url_for('.dashboard')
       },
       {
-        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
-        "label": "Apply to " + framework.name
+        "text": "Apply to " + framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_submission_lots", framework_slug=framework.slug),
-        "label": "Services"
-      },
-      {
-        "link": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot),
-        "label": service_data.lotName
-      } if not lot.oneServiceLimit else {}
+        "text": "Services",
+        "href": url_for(".framework_submission_lots", framework_slug=framework.slug)
+      }
     ]
   %}
-    {% include "toolkit/breadcrumb.html" %}
+    {% if lot.oneServiceLimit %}
+      {{ govukBreadcrumbs({"items": items + [{"text": section.name}]}) }}
+    {% else %}
+      {{ govukBreadcrumbs({"items": items + [
+        {
+          "text": service_data.lotName,
+          "href": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot)
+          },
+        {
+          "text": service_data.get('serviceName', service_data['lotName'])
+        }
+      ]}) }}
+    {% endif %}
```

Comment:
I don't think this is working, this is what I see when I visit http://localhost/suppliers/frameworks/digital-outcomes-and-specialists-4/submissions/digital-outcomes/150182

![image](https://user-images.githubusercontent.com/503614/71191551-8ae97200-227e-11ea-8c4b-5a95a5859239.png)

Reply:
This was caused by the wrong text value in DOS at line 30, replaced with `service_data.get('serviceName', service_data['lotName'])`.